### PR TITLE
Fix nightly compiler warnings

### PR DESCRIPTION
**Pull request by cmyr**
This fixes two small nightly warnings that were annoying me in CI.

### CODE REVIEW

File: druid/src/lib.rs [hunk 1]
```diff
@@ -127,7 +127,7 @@
 //! [`image` crate]: https://crates.io/crates/image
 
 #![deny(
-    broken_intra_doc_links,
+    rustdoc::broken_intra_doc_links,
```
Comment (Maan2003) [inline druid/src/lib.rs:130]: does not compile on rustc 1.51

Comment (cmyr) [reply druid/src/lib.rs:130]: Is this an issue? It does work on 1.52, and that's already two versions behind current..

Comment (Maan2003) [reply druid/src/lib.rs:130]: :shrug: I am not sure about bumping msrv just for this.

Comment (cmyr) [reply druid/src/lib.rs]: no sweat, happy to just merge the other one

**Review by Maan2003 — APPROVED** [review]

**Pull request merged**